R/testthat.R

```diff
@@ -94,14 +160,11 @@ test_single_database <- function(datasource, label = NULL, tests = "default") {
   )
 }
 
-testthat_database <- function(datasource, label = NULL, tests = "default") {
```

Comment:
Why did we remove the `label` arg?

Reply:
Because it wasn't being used.  Label is set in the parent function (`test_single_database`)